### PR TITLE
Improved fonts' RWD, improved tooltips

### PR DESCRIPTION
**Pull request by BartoszBilejczyk**
There are a few things that I need to talk with you about, bear that in mind pls - these are 

1) Model for ChartLabels (it's any now, because I had some issues there)
2) Updating chart series or asset id passes previous value, not the new one, I tried several things, but nothing seems to work, it's like Angular's (change)="" doesn't pass the right value.

### CODE REVIEW

File: src/app/common/components/chart/chart-settings.ts [hunk 1]
```diff
@@ -5,6 +5,91 @@ const chartOptions: ChartOptions = {
...
+      // Create element on first render
+      if (!tooltipEl) {
+        tooltipEl = document.createElement('div');
+        tooltipEl.id = 'chartjs-tooltip';
+        tooltipEl.innerHTML = '<table></table>';
+        document.body.appendChild(tooltipEl);
+      }
+
+      // Hide if no tooltip
+      if (tooltipModel.opacity == 0) {
```
Comment (KonradFak) [inline src/app/common/components/chart/chart-settings.ts]: Please use exact comparison by value and type

File: src/app/common/components/chart/chart-settings.ts [hunk 2]
```diff
@@ -5,6 +5,91 @@ const chartOptions: ChartOptions = {
   scaleShowVerticalLines: false,
   responsive: true,
   maintainAspectRatio: false,
+  tooltips: {
+    mode: 'index',
+    intersect: false,
+    enabled: false,
+    callbacks: {
+      title: function(tooltipItems, data) {
```
Comment (KonradFak) [inline src/app/common/components/chart/chart-settings.ts]: Would be good to add arguments types for functions in this file, also you should use es7 arrorw functions

File: src/app/common/components/chart/chart-settings.ts [hunk 3]
```diff
@@ -5,6 +5,91 @@ const chartOptions: ChartOptions = {
...
+          label += ': ';
+        }
+        label += Math.round(tooltipItem.yLabel * 100) / 100;
+        return label;
+      }
+    },
+
+    custom: function(tooltipModel) {
+      // Tooltip Element
+      let tooltipEl = document.getElementById('chartjs-tooltip');
```
Comment (KonradFak) [inline src/app/common/components/chart/chart-settings.ts]: Is it possible there would be multiple elements with this id generated in template? If so id is not good attribute for this as it should be unique

File: src/app/common/components/chart/chart-settings.ts [hunk 4]
```diff
@@ -5,6 +5,91 @@ const chartOptions: ChartOptions = {
...
+
+      // Create element on first render
+      if (!tooltipEl) {
+        tooltipEl = document.createElement('div');
+        tooltipEl.id = 'chartjs-tooltip';
+        tooltipEl.innerHTML = '<table></table>';
```
Comment (KonradFak) [inline src/app/common/components/chart/chart-settings.ts]: You can assign multiple fields at once:
```
Object.assign(tooltipEl, {
   id: 'chartjs-tooltip',
   innerHTML: '<table></table>'
});
```

File: src/app/common/components/chart/chart-settings.ts [hunk 5]
```diff
@@ -5,6 +5,91 @@ const chartOptions: ChartOptions = {
...
+        document.body.appendChild(tooltipEl);
+      }
+
+      // Hide if no tooltip
+      if (tooltipModel.opacity == 0) {
+        tooltipEl.style.opacity = '0';
+        return;
+      }
+
+      function getBody(bodyItem) {
```
Comment (KonradFak) [inline src/app/common/components/chart/chart-settings.ts]: Weird place to declare helper function, please move it out of custom field or at least move it on top of it

File: src/app/common/components/chart/chart-settings.ts [hunk 6]
```diff
@@ -5,6 +5,91 @@ const chartOptions: ChartOptions = {
...
+      tooltipEl.style.left = position.left + 15 + tooltipModel.caretX + 'px';
+      tooltipEl.style.top = position.top + window.pageYOffset + tooltipModel.caretY + 'px';
+      tooltipEl.style.textAlign = 'left';
+      tooltipEl.style.fontFamily = 'AvenirNext-Regular';
+      tooltipEl.style.fontSize = 13 + 'px';
+      tooltipEl.style.color = '#FFFFFF';
+      tooltipEl.style.padding = '7px 10px';
+      tooltipEl.style.borderRadius = '2px';
+      tooltipEl.style.boxShadow = '0 2px 8px 0 rgba(0, 0, 0, 0.15)';
+      tooltipEl.style.transition = 'opacity 0.3s ease';
```
Comment (KonradFak) [inline src/app/common/components/chart/chart-settings.ts]: You can do multiple assign at once

File: src/app/common/components/chart/chart-settings.ts (hunk 5 above)
Comment (BartoszBilejczyk) [reply src/app/common/components/chart/chart-settings.ts]: ok

File: src/app/common/components/chart/chart-settings.ts (hunk 6 above)
Comment (BartoszBilejczyk) [reply src/app/common/components/chart/chart-settings.ts]: ok

File: src/app/common/components/chart/chart-settings.ts (hunk 4 above)
Comment (BartoszBilejczyk) [reply src/app/common/components/chart/chart-settings.ts]: ok

File: src/app/common/components/chart/chart-settings.ts (hunk 1 above)
Comment (BartoszBilejczyk) [reply src/app/common/components/chart/chart-settings.ts]: ok

**Pull request merged**